library/modules/bigip_provision.py

```diff
@@ -234,19 +310,26 @@ def exec_module(self):
             changed = self.present()
         elif state == "absent":
             changed = self.absent()
-
-        changes = self.changes.to_return()
```

Comment:
why this change ?

Reply:
Module did not use to have reportable changes but now does. Modified changes variable to now run through ReportableChanges function.